test/sync-test.ts

```diff
@@ -65,85 +65,90 @@ describe('napajs/sync', function () {
     it('@napa: sync.Lock - multi thread sync execution (not using lock)', () => {
         let napaZone = napa.zone.create('zone-for-sync-test-1', { workers: 2 });
         napaZone.broadcast(spinWait.toString());
-        napaZone.broadcast("require('../lib/index')");
+        napaZone.broadcast("require('../lib/index');");
```

Comment:
BTW, why do we need this line?

Reply:
I want to use this line to reduce the time of next `require` call.

Reply:
You don't need this, since during all napa.zone.create, `require('napajs')` is broadcasted to all workers automatically.